Cannot install firebase-tools with Node 8 since v7.8.0

# Description of the issue

Running the following command with Node 8 (using `nvm`, `lts/carbon`):
```
npm install -g firebase-tools
```
Throws the following error:
```
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for semver@7.1.1: wanted: {"node":">=10"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm ERR! notsup Not compatible with your version of node/npm: semver@7.1.1
npm ERR! notsup Not compatible with your version of node/npm: semver@7.1.1
npm ERR! notsup Required: {"node":">=10"}
npm ERR! notsup Actual:   {"npm":"6.13.4","node":"8.17.0"}
```

# Additional information
I've tried installing previous versions, and the error persists until `firebase-tools@7.7.0`.
The npm debug log show the following relevant information:
```
682 silly resolveWithNewModule ecdsa-sig-formatter@1.0.11 checking installable status
683 http fetch GET 304 https://registry.npmjs.org/yallist 139ms (from cache)
684 silly pacote range manifest for yallist@^3.0.2 fetched in 146ms
685 silly resolveWithNewModule yallist@3.1.1 checking installable status
686 silly pacote range manifest for semver@^7.0.0 fetched in 176ms
687 silly resolveWithNewModule semver@7.1.1 checking installable status
688 http fetch GET 304 https://registry.npmjs.org/@grpc%2fgrpc-js 223ms (from cache)
689 http fetch GET 304 https://registry.npmjs.org/@grpc%2fproto-loader 224ms (from cache)
690 http fetch GET 304 https://registry.npmjs.org/lodash.has 230ms (from cache)
691 silly pacote range manifest for @grpc/grpc-js@^0.6.12 fetched in 233ms
692 silly resolveWithNewModule @grpc/grpc-js@0.6.15 checking installable status
693 http fetch GET 304 https://registry.npmjs.org/@types%2ffs-extra 233ms (from cache)
694 silly pacote range manifest for @grpc/proto-loader@^0.5.1 fetched in 235ms
695 silly resolveWithNewModule @grpc/proto-loader@0.5.3 checking installable status
696 silly pacote range manifest for lodash.has@^4.5.2 fetched in 234ms
697 silly resolveWithNewModule lodash.has@4.5.2 checking installable status
698 silly pacote range manifest for @types/fs-extra@^8.0.1 fetched in 236ms
699 silly resolveWithNewModule @types/fs-extra@8.0.1 checking installable status
700 http fetch GET 304 https://registry.npmjs.org/duplexify 236ms (from cache)
701 http fetch GET 304 https://registry.npmjs.org/lodash.at 276ms (from cache)
702 silly pacote range manifest for duplexify@^3.6.0 fetched in 301ms
703 silly resolveWithNewModule duplexify@3.7.1 checking installable status
704 silly pacote range manifest for lodash.at@^4.6.0 fetched in 357ms
705 silly resolveWithNewModule lodash.at@4.6.0 checking installable status
706 http fetch GET 304 https://registry.npmjs.org/retry-request 368ms (from cache)
707 silly pacote range manifest for retry-request@^4.0.0 fetched in 369ms
708 silly resolveWithNewModule retry-request@4.1.1 checking installable status
709 http fetch GET 304 https://registry.npmjs.org/walkdir 982ms (from cache)
710 silly pacote range manifest for walkdir@^0.4.0 fetched in 983ms
711 silly resolveWithNewModule walkdir@0.4.1 checking installable status
712 timing stage:rollbackFailedOptional Completed in 1ms
713 timing stage:runTopLevelLifecycles Completed in 14650ms
714 verbose stack Error: Unsupported engine for semver@7.1.1: wanted: {"node":">=10"} (current: {"node":"8.17.0","npm":"6.13.4"})
```
The culprit seems to be one `semver@^7.0.0` dependency (line 686), but I can't seem to find where it is declared.
# Environment

```
OS Ubuntu 18.04.3 LTS
nvm 0.34.0
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,0 +1,1 @@
+* Set a dependency (`google-gax`) so that the CLI coontinues to install successfully on Node 8 environments.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,1 +1,1 @@
-* Set a dependency (`google-gax`) so that the CLI coontinues to install successfully on Node 8 environments.
+* Set a dependency (`google-gax`) so that the CLI continues to install successfully on Node 8 environments.

--- a/package.json
+++ b/package.json
@@ -84,6 +84,7 @@
     "fs-extra": "^0.23.1",
     "glob": "^7.1.2",
     "google-auto-auth": "^0.7.2",
+    "google-gax": "~1.12.0",
     "inquirer": "~6.3.1",
     "jsonschema": "^1.0.2",
     "jsonwebtoken": "^8.2.1",

